### ISSUE TITLE
Asking for code component

### ISSUE THREAD

**Issue by cenchaojun** [SysCV/transfiner#40]
thank you for your nice work. I want to use these modules for salient object detection. I can't find the relevant code for the paper. Would you mind showing me where is the code for this component below?
![image](https://user-images.githubusercontent.com/48192514/192293740-397375bc-04e4-4d3f-a51a-041b8ba1c6ec.png)
besh wish to you.❤️

**Comment by lkeab** [comment]
Hi please refer to the discussion [here](https://github.com/SysCV/transfiner/issues/20).

**Comment by cenchaojun** [comment]
> Hi please refer to the discussion [here](https://github.com/SysCV/transfiner/issues/20).

yes. I have seen this link. But, I still can't find some modules codes like the Point Quadtree and Coarse Masks ROI Feature

Thank you for your quick reply

**Comment by Liu233QAQ** [comment]
I met the same problem, have you found the Incoherence Detector module and Point Quadtree now?
Thanks in advance!
